pkg/sink/sink.go

```diff
@@ -145,10 +145,26 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 		}
 	}
 
-	// TODO: Do we really need to return the entire body back???
+	js, err := json.Marshal(
+		struct {
+			EventListener string
+			Namespace     string
+			EventID       string
+		}{
+			r.EventListenerName,
+			r.EventListenerNamespace,
+			eventID,
+		},
+	)
+	if err != nil {
+		http.Error(response, "Error Marshaling response"+
+			err.Error(), http.StatusInternalServerError)
+		return
+	}
+
 	response.WriteHeader(code)
-	fmt.Fprintf(response, "EventListener: %s in Namespace: %s handling event (EventID: %s) with payload: %s and header: %v",
```

Comment:
Make sure this (or something equivalent) is logged somewhere. This is useful for debugging whether requests were actually received or not.

Reply:
Yeah I think we already do this.

Reply:
Looks like its Debugf: https://github.com/tektoncd/triggers/blob/master/pkg/sink/sink.go#L83

Might be worth logging the eventID or something in Info and the body in Debug